### PR TITLE
7708-onboarding-flow-improvements-desktop

### LINKED ISSUE #7833: Change YOUR ACCOUNT to say TRADING ACCOUNT in dropdown

### PR DESCRIPTION
**Pull request by andrewdamelio**
Improvements to onboarding flow #7708

closes https://github.com/AugurProject/augur/issues/7833 https://github.com/AugurProject/augur/issues/7832

```User has signed up but needs to add funds```

```User has added funds but needs to activate```

### CODE REVIEW

File: packages/augur-ui/src/modules/modal/common.tsx [hunk 1]
```diff
@@ -406,20 +410,38 @@ interface TransferMyDaiProps {
...
+        />
+      </div>
+    );
+  }
+
+  return (
+    <div className={Styles.TransferMyDai}>
+      <div>
+        <span>{daiAmount.formattedValue} Dai in your {walletType} wallet</span>
+        <span>Transfer any amount to your Augur user account. </span>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/common.tsx]: Don't use `Augur user account` use `trading account`

File: packages/augur-ui/src/modules/modal/transfer-form.tsx [hunk 2]
```diff
@@ -113,14 +121,24 @@ export class TransferForm extends Component<
...
     const newAmount = convertExponentialToDecimal(sanitizeArg(amount));
     const bnNewAmount = createBigNumber(newAmount || '0');
     const { errors: updatedErrors, currency } = this.state;
     updatedErrors.amount = '';
-    const availableEth = createBigNumber(balances.eth);
-    const availableDai = createBigNumber(balances.dai);
+    let availableEth = createBigNumber(balances.eth);
+    let availableDai = createBigNumber(balances.dai);
+    const balance = useSigner
+      ? balances[`${currency.toLowerCase()}NonSafe`]
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/transfer-form.tsx]: not a fan of this notation hard to find when searching for usages of loginAccount.balances.XXX

File: packages/augur-ui/src/modules/modal/transfer-form.tsx [hunk 3]
```diff
@@ -92,15 +96,19 @@ export class TransferForm extends Component<
   };
 
   handleMax = () => {
-    const { balances, fallBackGasCosts, GsnEnabled, gasPrice } = this.props;
+    const { balances, useSigner, fallBackGasCosts, GsnEnabled, gasPrice } = this.props;
     const { currency, relayerGasCosts } = this.state;
 
+    const balance = useSigner
+      ? balances[`${currency.toLowerCase()}NonSafe`]
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/transfer-form.tsx]: not a fan of composite string keys, hard to search and find usages when refactoring use the dot notation balances.XXX,

**Review by bthaile — CHANGES_REQUESTED** [review]
minor code changes needed

**Comment by bthaile** [comment]
user doesn't have REP in their signing wallet or any ETH. only DAI. Shouldn't show other currencies in dropdown and should only show DAI and should be readonly

image

If user wants to transfer all DAI from signing wallet to trading account and clicks MAX and doesn't have any ETH in their signing wallet, we need to remove estimated fees from total. We'll need to use the relayer to transfer DAI from signing wallet to trading account.

image

user doesn't have ETH in signing wallet
image

**Comment by bthaile** [comment]
If user opens up `transfer my dai` modal from account dropdown then closes the `transfer my dai` modal user shouldn't be sent to on-boarding modal. Should just close the `transfer my dai` modal.

**Comment by bthaile** [comment]
fixed merge conflict. fixed flash method to set liquidity to 200 DAI / ETH. can be used for kovan as well.

File: packages/augur-ui/src/modules/modal/transfer-form.tsx (hunk 2 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/modules/modal/transfer-form.tsx]: will replace 👍

**Comment by andrewdamelio** [comment]
@bthaile all changes made except the below comment -- need more time to look into this but lets move ahead with this PR as is so product can start testing/feedbacking.

> If user wants to transfer all DAI from signing wallet to trading account and clicks MAX and doesn't have any ETH in their signing wallet, we need to remove estimated fees from total. **We'll need to use the relayer to transfer DAI from signing wallet to trading account.**

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**